pypeit/scripts/view_fits.py

```diff
@@ -29,9 +29,11 @@ def get_parser(cls, width=None):
         parser.add_argument('--exten', type=int, default=None,
                             help='Show a FITS extension in the raw file. Note --proc and --mosaic '
                                  'will not work with this option.')
-        parser.add_argument('--det', type=str, default=1,
-                            help='Detector number. To mosaic keck_deimos or keck_lris images, '
-                                 'set equal to mosaic.')
+        parser.add_argument('--det', type=str, default='1', nargs='*',
```

Comment:
Can you make sure that this works in case: 

1. View the raw detector image with no processing, let's call that det and 1 as an example. 
2. View an allowed mosaic for an instrument that supports that with --mosaic option. In this case we need to think about which mosaic to display
3. --proc vs not --proc. I think mosaic requires --proc but it may not require all of the steps. Need to think through that.